--- gir-repository-0.6.3/gir/Makefile.am~	2009-06-21 23:40:37.000000000 +0200
+++ gir-repository-0.6.3/gir/Makefile.am	2009-07-06 19:22:45.025148943 +0200
@@ -47,7 +47,7 @@
 endif
 
 if BUILD_GCONF
-GCONF_INCLUDEDIR=`pkg-config --variable=includedir gconf-2.0`/gconf/2
+GCONF_INCLUDEDIR=`pkg-config --variable=includedir gconf-2.0`/gconf2
 GConf-2.0.gir: $(INTROSPECTION_SCANNER)
 	$(INTROSPECTION_SCANNER) -v --namespace GConf --nsversion=2.0 \
 	    --add-include-path=$(srcdir) --add-include-path=. \
